Global/XilinxISE.gitignore

```diff
@@ -65,3 +69,9 @@ xlnx_auto_0_xdb/
 xst/
 _ngo/
 _xmsgs/
+work/
+
+#ignore OS noise
```

Comment:
OS junk files are captured in the existing `Global/macOS.gitignore` - please remove them from this template